src/modalities/trainer.py

```diff
@@ -260,9 +260,9 @@ def train(
                     metrics=metrics,
                     # TODO: hardcoded metric key
                     throughput_metrics={
-                        "train samples/s": synced_num_samples_per_second,
-                        "train mfu": mfu,
-                        "lr mean": torch.tensor(scheduler.get_last_lr()).mean(),
+                        "train samples/s": ResultItem(synced_num_samples_per_second, 1),
+                        "train mfu": ResultItem(mfu, 2),
+                        "lr mean": ResultItem(torch.tensor(scheduler.get_last_lr()).mean(), 6),
```

Comment:
Since the learning rate can be arbitrarily small, I would log it in full precision.

Reply:
done